documentation/API_documentation/QoD_API.md

```diff
@@ -308,6 +307,177 @@ Following table defines API endpoints of exposed REST based for QoD management o
+        <tr>
+            <td><b>503: Service temporarily unavailable</b></td>
+        </tr>
+    </tbody>
+</table>
+
+#### QoD - Get QoS Profiles Resource by id
+
+<table>
+    <thead>
+        <tr>
+            <th colspan=3><b>Get QoS Profile Resource by id</b></th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><b>HTTP Request</b></td>
+            <td colspan=2>GET &lt;base-url&gt;/qod/v0/qosProfile</td>
+        </tr>
+        <tr>
+            <td><b>Query Parameters</b></td>
+            <td colspan=2> No Query Parameters</td>
+        </tr>
+        <tr>
+            <td><b>Path Parameters</b></td>
+            <td><b>profile-id</b></td>
+            <td>Extended QoS profile id</td>
+        </tr>
+        <tr>
+            <td><b>Request Body Parameters</b></td>
+            <td colspan=2><b>No Request Body Parameters defined</b></td>
+        </tr>
+        <tr>
+            <td rowspan=7><b>Response</b></td>
+            <td><b>201: QoS Profile created</b></td>
+            <td>
+                Response body:<br>
+                <b>id</b>: The identifier of the QoS Profile<br>
+                <b>name</b>: The name of the QoS Profile<br>
+                <b>status</b>: The status of the QoS Profile<br>
+                <b>targetMinimumUpstreamRate</b>: This is the targeted minimum upstream rate for this profile.<br>
+                <b>maxUpstreamRate</b>: This is the maximum sustained upstream rate for this profile. If this is undefined, then the maxUpstream rate will not change.
+                If this rate is lower than the current provisioned maximum sustained rate, then the current rate is used.<br>
+                <b>maxUpstreamBurstRate</b>: This is the maximum burst upstream rate for this profile. If this is undefined, then the maxUpstream rate will not change.
+                If this rate is lower than the current provisioned maximum rate, then the current maximum burst rate is used.<br>
+                <b>targetMinimumDownstreamRate</b>: This is the targeted minimum Downstream rate for this profile.<br>
+                <b>maxDownstreamRate</b>: This is the maximum sustained Downstream rate for this profile. If this is undefined, then the maxDownstream rate will not change.
+                If this rate is lower than the current provisioned maximum sustained rate, then the current rate is used.<br>
+                <b>maxDownstreamBurstRate</b>: This is the maximum burst Downstream rate for this profile. If this is undefined, then the maxDownstream rate will not change.
+                If this rate is lower than the current provisioned maximum rate, then the current maximum burst rate is used.<br>
+                <b>minDuration (optional)</b>: Minimum duration for a session with this QoS Profile<br>
+                <b>maxDuration (optional)</b>: Maximum duration for a session with this QoS Profile<br>
+                <b>priority (optional)</b>: The priority of this QoS Profile<br>
```

Comment:
<b>priority (optional)</b>:  The priority that will be given to sessions using this QoS profile, where a lower value indicates a higher priority